### PR TITLE
PLAT-139226: Update interoperability.md for Sandstone

### PR DESCRIPTION
**Pull request by YunhoLee0**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Currently, interoperability.md is based on Moonstone.
It needs to be updated for Sandstone.

### Resolution
Update descriptions and example codes for Sandstone.

### Additional Considerations
There are template codes linked with the codesandbox.
Currently, the template in the codesandbox is owned by my account.
I'm considering how to make it publicly owned.

### Links
PLAT-139226

### CODE REVIEW

File: docs/interoperability.md [hunk 1]
```diff
@@ -106,53 +108,50 @@ is a good reference for enabling CSS modules prior to `2.0.0`.
...
 "devDependencies" : {
-  "@enact/dev-utils": "2.0.0"
+  "@enact/dev-utils": "^4.1.1"
 },
 "dependencies": {
   ...
-  "@enact/i18n": "2.3.0",
-  "@enact/moonstone": "2.3.0",
+  "@enact/i18n": "^4.0.2",
+  "@enact/sandstone": "^1.5.1",
```
Comment (MikyungKim) [inline docs/interoperability.md:149]:
```suggestion
  "@enact/sandstone": "^2.0.0",
```

**Review by MikyungKim — APPROVED** [review]
LGTM

File: docs/interoperability.md (hunk 1 above)
Comment (YunhoLee0) [reply docs/interoperability.md]: It's fixed.
I agree it would be better to use 2.0.0 for later.

**Review by seunghoh — APPROVED** [review]
LGTM

**Pull request merged**